build.gradle

```diff
@@ -73,6 +73,10 @@ subprojects {
                 classpath('com.google.protobuf:protobuf-java:[3.25.5,)') {
                     because 'CVE-2024-7254 is fixed in 3.25.5 and above'
                 }
+
+                classpath('commons-io:commons-io:[2.14.0,)') {
```

Comment:
You don't need this one unless it's brought in by a plugin, rather than a dependency